src/Microsoft.DotNet.Cli.Utils/Product.cs

```diff
@@ -1,22 +1,17 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.Reflection;
-
 namespace Microsoft.DotNet.Cli.Utils
 {
     public class Product
     {
-        public static string LongName => LocalizableStrings.DotNetCommandLineTools;
+        public static string LongName => LocalizableStrings.DotNetSdkInfo;
```

Comment:
Why are we changing this? Was that decided? It seems unrelated to me.

Reply:
I think the change from ".NET Command Line Tools to .NET Core SDK" is a good one. But it's a separate issue. When we do this change we should also delete DotNetCommandLineTools. I'd prefer a separate PR, but if you want to keep this, please also delete the old string so that it's easier to see what is actually changing in the text.

Reply:
@nguerrera 
Old string is now deleted.